File: calculator.java
import java.util.Scanner;

public class calculator {
    public static void main(String[] agrs) {
        Scanner Input = new Scanner(System.in);

        System.out.print("\nEnter the first number: ");
        double number1 = Input.nextDouble();

        System.out.print("Enter the second number: ");
        double number2 = Input.nextDouble();

        System.out.print(
                "\n1. Addition\n2. Subtraction\n3. Multiplication\n4. Division\nEnter the operation number would you like to perform: ");
        int operation = Math.abs(Input.nextInt());

        Input.close();

        switch (operation) {
            case 1:
                add(number1, number2);
                break;
            case 2:
                sub(number1, number2);
                break;
            case 3:
                mul(number1, number2);
                break;
            case 4:
                div(number1, number2);
                break;
            default:
                System.out.println("\n" + operation + " is an invalid operation...");
                break;
        }
    }

    public static void add(double num1, double num2) {
        double answer = num1 + num2;
        System.out.print("\nThe sum of " + num1 + " and " + num2 + " is " + answer + "!");
    }

    public static void sub(double num1, double num2) {
        double answer = num1 - num2;
        System.out.print("\nThe difference of " + num1 + " and " + num2 + " is " + answer + "!");
    }

    public static void mul(double num1, double num2) {
        double answer = num1 * num2;
        System.out.print("\nThe product of " + num1 + " and " + num2 + " is " + answer + "!");
    }

    public static void div(double num1, double num2) {
        double answer = num1 / num2;
        System.out.print("\nThe quotient of " + num1 + " and " + num2 + " is " + answer + "!");
    }
}

File: calculator.cpp
#include <iostream>

class Calculator
{
public:

	double number1;
	double number2;
	int operation;

	void add(double num1, double num2) // Function for addition.
	{
		double answer = num1 + num2;
		std::cout << "\nThe sum of " << num1 << " and " << num2 << " is " << answer << "!";
	}

	void subtract(double num1, double num2) // Function for subtraction.
	{
		double answer = num1 - num2;
		std::cout << "\nThe difference of " << num1 << " and " << num2 << " is " << answer << "!";
	}

	void multiply(double num1, double num2) // Function for mutiplication.
	{
		double answer = num1 * num2;
		std::cout << "\nThe product of " << num1 << " and " << num2 << " is " << answer << "!";
	}

	void divide(double num1, double num2) // Function for division.
	{
		double answer = num1 / num2;
		std::cout << "\nThe difference of " << num1 << " and " << num2 << " is " << answer << "!";
	}

	void calculate(double number1, double number2, int operation) {

		Calculator Calculator;

		switch (operation) // Switch to run the correct operation type. (If operation > 4, defaults to "invalid")
		{
		case 1:
			Calculator.add(number1, number2);
			break;
		case 2:
			Calculator.subtract(number1, number2);
			break;
		case 3:
			Calculator.multiply(number1, number2);
			break;
		case 4:
			Calculator.divide(number1, number2);
			break;
		default:
			std::cout << operation << " is an invalid opertation..." << std::endl;
			break;
		}

	}
};

int main() // Runs the main function of the calculator.
{
	Calculator Calculator;

	Calculator.number1; // Input for the first number.
	std::cout << "Enter the first number: ";
	std::cin >> Calculator.number1;

	Calculator.number2; // Input for the second number.
	std::cout << "Enter the second number: ";
	std::cin >> Calculator.number2;

	Calculator.operation; // Input for the operation type.
	std::cout << "\n1. Addition\n2. Subtraction\n3. Multiplication\n4. Division\nEnter the operation number would you like to perform: ";
	std::cin >> Calculator.operation;

	Calculator.calculate(Calculator.number1, Calculator.number2, Calculator.operation);
}
